### DISCUSSION TITLE
awesome project, can produce web deliverables out of the box

### DISCUSSION THREAD

**Discussion by pmp-p** [BirdLogics/sb3topy#11] [Show and tell]
sb3topy + add custom main.py in the generated folder -> pygbag (pygame-ce) -> itch -> https://pmp-p.itch.io/opac3tyd

typical main.py
```py
# /// pyproject
# [project]
# name = "name"
# version = "version"
# description = "description"
# readme = {file = "README.txt", content-type = "text/markdown"}
# requires-python = ">=3.11"
#
# dependencies = [
#     "pygame-ce",
# ]
# ///

import pygbag.aio as asyncio
import os
from pathlib import Path
from project import *
os.chdir( Path(engine.__file__ ).parent.parent )
runtime = engine.runtime.Runtime(engine.events.SPRITES)
asyncio.run(runtime.main_loop())
```

**Comment by BirdLogics** [comment]
Neat, that's pretty cool. Personally, I think that the [TurboWarp Packager](https://packager.turbowarp.org/) is better suited to this, since it uses JavaScript instead of Python. Running Pygame on the web seems a bit janky. It is still pretty neat that it works though!
